core/modules/admin_bar/css/admin_bar.css

```diff
@@ -360,11 +217,25 @@
 
 /* Second-and-more-level hovering. */
 #admin-bar .dropdown li li.expandable > a {
-  background: #45454A url(../images/arrow.png) no-repeat 145px center; /* LTR */
+  background: #45454A;
+}
+#admin-bar .dropdown li li.expandable > a::before {
+  content: '';
+  /*noinspection CssUnresolvedCustomProperty*/
+  mask-image: var(--icon-arrow-square-right); /* LTR */
+  mask-repeat: no-repeat;
+  background: #FFF;
```

Comment:
Shouldn't hex values be lowercase? And only three... (I remember some discussion re hex color values in B, but can't remember the context...)

Reply:
Oh, here it is: https://docs.backdropcms.org/css-standards#format

"When hex values are used for colors, use lowercase and the long syntax, e.g. #aaaaaa."

Reply:
Thanks, all of admin_bar.css could use a refresh it seems.